arlo-client/src/setupProxy.js

```diff
@@ -15,6 +15,7 @@ module.exports = function(app) {
   app.use(proxy('/admin', { target }))
   app.use(proxy('/election/new', { target }))
   app.use(proxy('/election/*/audit/**', { target }))
+  app.use(proxy('/election/*/settings', { target }))
   app.use(proxy('/election/*/jurisdiction/**', { target }))
   app.use(proxy('/election/*/jurisdictions/**', { target }))
   app.use(proxy('/election/*/admin/**', { target }))
```

Comment:
is there any reason all of these aren't just `/election/*`?

Reply:
It breaks development, since it sends e.g. `/election/<election_id>` to the python app which then serves the production-built React app rather than the development one.